packages/page-template-modal/package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "@automattic/page-template-modal",
+	"version": "1.0.0",
+	"description": "Automattic Page Template Modal",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"calypso:src": "src/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/page-template-modal"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"dependencies": {
+		"@wordpress/api-fetch": "^3.3.0",
+		"@wordpress/block-editor": "^5.2.1",
+		"@wordpress/blocks": "^6.25.1",
+		"@wordpress/components": "^12.0.1",
+		"@wordpress/compose": "^3.23.1",
+		"@wordpress/data": "^4.26.1",
+		"@wordpress/editor": "^9.25",
+		"@wordpress/element": "^2.19.01",
+		"@wordpress/i18n": "^3.17.0",
+		"@wordpress/nux": "^3.24.1",
+		"@wordpress/url": "^2.21.0",
+		"classnames": "^2.2.6",
+		"lodash": "^4.17.19"
+	},
+	"devDependencies": {
+		"@automattic/calypso-build": "^7.0.0"
+	},
+	"peerDependencies": {
+		"react": "^16.8"
+	},
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"build": "calypso-build",
```

Comment:
I think this should be `transpile` instead of `calypso-build`? `transpile` is what other packages use, it produces the dirs `dist/esm` and `dist/cjs` (declared above in this `package.json`) and you won't need a custom `wepbkac.config.js`

Reply:
this was to make sure that the scss file gets built and output as css in `dist`.

When we use this package form the ETK, we don't actually use the built css from `dist`, rather, we import the `scss` file directly, so maybe it doesn't matter? 

But yea, the motivation for using `calypso-build` was so that the css gets built too

Reply:
little bit of background up discussion with @p-jackson p1613363841195300-slack-CRWCHQGUB